app/controllers/request_purpose.js

```diff
@@ -87,7 +79,7 @@ export default Ember.Controller.extend(cancelOrder, {
       let orderParams = {
         organisation_id: user_organisation_id,
         purpose_description: this.get('description'),
-        purpose_ids: purposeIds,
+        purpose_ids: [],
```

Comment:
@patrixr As we have removed hard-coded values which are good. How we are sending purpose_ids to API?  Might Sound like a silly question but I think I am missing something.

Reply:
@namrataukirde , that is now done on the following page (client_information).
The radio buttons and logic have been moved there by señor @abulsayyad123  :D

--> https://github.com/crossroads/browse.goodcity/blob/master/app/controllers/order/client_information.js#L83

Reply:
That's Great. Thanks @patrixr , @abulsayyad123 :)